prime-angular-frontend/src/app/modules/sat-eforms/shared/modules/sat-eforms-login-page/sat-eforms-login-page.component.ts

```diff
@@ -24,7 +24,7 @@ export class SatEformsLoginPageComponent implements OnInit {
     private authService: AuthService,
   ) {
     this.title = 'Enrol for access to Special Authority eForms';
-    this.loginLabel = 'Enrol using your BCSC';
+    this.loginLabel = 'Enrol using your BC Service Card';
```

Comment:
That should be "BC Services Card" (https://www2.gov.bc.ca/gov/content/governments/government-id/bc-services-card)

Reply:
I had it as **Services** but then looked at the ticket and it said **Service** 😆 , will change it back.